.github/workflows/simpletest-db-config.yml

```diff
@@ -0,0 +1,41 @@
+name: SimpleTest (Database Config)
+on:
+  pull_request:
+    branches: [1.*]
+jobs:
+  simpletest-db-config:
+    name: SimpleTest
+    runs-on: ubuntu-22.04
+    # Skip this job if the PR title contains "[skip tests]".
+    if: ${{ !contains(github.event.pull_request.title, '[skip tests]') }}
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['8.3']
+        database-versions: ['mariadb-10.3']
```

Comment:
I wonder, if it's time to switch to a newer mariadb version, as 10.3 is quite dated. But we could discuss that also in a follow-up (actually unrelated here).